src/otx/core/model/segmentation.py

```diff
@@ -37,14 +41,22 @@
 class OTXSegmentationModel(OTXModel[SegBatchDataEntity, SegBatchPredEntity]):
     """Base class for the semantic segmentation models used in OTX."""
 
+    mean: ClassVar[tuple[float, float, float]] = (0.485, 0.456, 0.406)
+    scale: ClassVar[tuple[float, float, float]] = (0.229, 0.224, 0.225)
+
     def __init__(
         self,
         label_info: LabelInfoTypes,
-        input_size: tuple[int, int],
+        input_size: tuple[int, int] = (512, 512),
         optimizer: OptimizerCallable = DefaultOptimizerCallable,
         scheduler: LRSchedulerCallable | LRSchedulerListCallable = DefaultSchedulerCallable,
         metric: MetricCallable = SegmCallable,  # type: ignore[assignment]
         torch_compile: bool = False,
+        train_type: Literal[OTXTrainType.SUPERVISED, OTXTrainType.SEMI_SUPERVISED] = OTXTrainType.SUPERVISED,
+        model_version: str | None = None,
```

Comment:
Why is `None` needed as a default? Is it a required parameter?

Reply:
Made it required parameter https://github.com/openvinotoolkit/training_extensions/pull/3863